audit-service/pom.xml

```diff
@@ -92,12 +92,12 @@
             <artifactId>akka-discovery_${scala.version}</artifactId>
             <version>${akka.version}</version>
         </dependency>
-        <dependency>
-            <!-- Stroom -->
+        <!--<dependency>
```

Comment:
Add stroom back in but use 
<dependency>
  <groupId>uk.gov.gchq.eventlogging</groupId>
  <artifactId>event-logging</artifactId>
  <version>4.0.8_schema-v3.2.4</version>
</dependency>
rather than the dependancy from bintray which will disappear at the end of the month